content/applications/inventory_and_mrp/maintenance/maintenance_setup.rst

```diff
@@ -0,0 +1,239 @@
+
+In the :guilabel:`Cost` field, specify how much the equipment cost to acquire, if applicable.
+
+If the equipment is covered under a warranty, specify the :guilabel:`Warranty Expiration Date` by
+selecting a date from the calendar popover in that field.
+
+Maintenance metrics
+~~~~~~~~~~~~~~~~~~~
+
+Various maintenance metrics are available for each piece of equipment, and are automatically
+computed based on corrective maintenance, and planned preventive maintenance.
+
+To view the maintenance metrics for a specific piece of equipment, from the equipment form, click
+the :guilabel:`Maintenance` tab.
+
+.. image:: maintenance_setup/maintenance-setup-metrics.png
+   :align: center
+   :alt: Maintenance tab on equipment form showing computed metrics fields.
+
+Doing so reveals the following fields:
+
+- :guilabel:`Expected Mean Time Between Failure`: the amount of time (in days) before the next
+  failure is expected. This is the **only** field not greyed-out, and the **only** field users can
+  edit.
+- :guilabel:`Mean Time Between Failure`: the amount of time (in days) between reported failures.
+  This value is computed based on completed corrective maintenances.
+- :guilabel:`Estimated Next Failure`: the date on which the next failure is expected. This date is
+  computed as the Latest Failure Date + |MTBF|.
+- :guilabel:`Latest Failure`: The date of the latest failure. The value in this field updates once a
+  failure is reported for this equipment.
+- :guilabel:`Mean Time To Repair`: the amount of time (in days) it takes to repair this equipment
+  upon failure. This value updates once a maintenance request is completed for this equipment.
+
+Work centers
+============
+
+To view the work centers where equipment is being used, and how the equipment is being used in them,
+navigate to :menuselection:`Maintenance app --> Equipment --> Work Centers`, and click into a work
+center.
+
+From the resulting work center form, click the :guilabel:`Equipment` tab to view all machines and
+tools being used in that specific work center.
+
+Each piece of equipment is listed with certain relevant information: the :guilabel:`Equipment Name`,
+the responsible :guilabel:`Technician`, the :guilabel:`Equipment Category` it belongs to, and a few
+important maintenance metrics: its |MTBF|, |MTTR|, and :guilabel:`Est. Next Failure` date.
+
+.. image:: maintenance_setup/maintenance-setup-work-center.png
+   :align: center
+   :alt: List of equipment included in a work center.
+
+.. tip::
+   To add new equipment to a work center directly from the work center form, click :guilabel:`Add a
+   line` under the :guilabel:`Equipment` tab. This opens an :guilabel:`Add: Maintenance Equipment`
+   pop-up window.
+
+   From the pop-up window, select the equipment that should be added to the work center, and click
+   :guilabel:`Select`. Click :guilabel:`Save` to save changes.
```

Comment:
again, is clicking 'save' totally necessary here, with the auto-save feature?

Reply:
No, removing that sentence - Suggestion implemented ✅